src/librustc_back/target/mod.rs

```diff
@@ -359,6 +359,11 @@ pub struct TargetOptions {
     // will 'just work'.
     pub obj_is_bitcode: bool,
 
+    // LLVM can't produce object files for this target. Instead, we'll make LLVM
+    // emit assembly and then use `gcc` to turn that assembly into an object
+    // file
+    pub no_integrated_as: bool,
```

Comment:
This ... could have a more descriptive name I supposed. I just use the same name as the `-C no-integrated-as` flag.